framework/doc/content/documentation/application_development/performance_benchmarking.md

```diff
@@ -1,41 +1,91 @@
 
```

Comment:
You need to get rid of this empty line. This is a quirk of my system that I have not yet resolved. The top level heading must be at the very top. Something I need to fix, I know the cause, just haven't got to it yet.